examples/deck/custom-layers/outlined-geojson-layer.js

```diff
@@ -0,0 +1,54 @@
+// @flow
+import { CompositeLayer, GeoJsonLayer } from 'deck.gl';
+
+export default class OutlinedGeoJsonLayer extends CompositeLayer {
```

Comment:
The outlined layers don't really have anything to do with the layer overrides. They are just new layers, right.

Reply:
Correct, they served as examples mostly.